build/themes/default.html

```diff
@@ -495,6 +494,15 @@
     top: 0.125em;
   }
 
+  /* -------------------------------------------------- */
+  /* references */
+  /* -------------------------------------------------- */
+
+  .csl-entry {
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
```

Comment:
I think I'd be in favor of a slightly smaller margin between references than:

![image](https://user-images.githubusercontent.com/1117703/120576437-c0d9d200-c3f0-11eb-8f2a-a014b073d575.png)

I think we do want some, but since reference sections can get quite long, perhaps we could get by with slightly less? Maybe 15 (or 50% to 75% of the current spacing)?

Reply:
Reduced to 15px. I wouldn't recommend going lower than that, as it becomes difficult to distinguish for sighted and hard-of-seeing people alike.

Reply:
Here's 15. Looks good:

![image](https://user-images.githubusercontent.com/1117703/120678933-a2162280-c466-11eb-87c5-91d2467e1e25.png)